easybuild/toolchains/linalg/openblas.py

```diff
@@ -40,6 +40,7 @@ class OpenBLAS(LinAlg):
     """
     BLAS_MODULE_NAME = ['OpenBLAS']
     BLAS_LIB = ['openblas']
+    BLAS_LIB_MT = ['openblas']
```

Comment:
This doesn't actually add -lpthread. And the libopenblas.{a,so} we build actually depends on -lpthread even for the non-MT version since there is only one lib and it's built with OpenMP enabled.

Reply:
> This doesn't actually add -lpthread.

yes it does. did you actually check it? it’s because `BLAS_LIB_MT` was not defined that `-lpthread` was not added to `LIBBLAS_MT`. defining it here fixes it (for me at least).

> And the libopenblas.{a,so} we build actually depends on -lpthread even for the non-MT version since there is only one lib and it’s built with OpenMP enabled.

that’s true for the OpenBLAS versions in the official EB repo but people might still want to build it without OpenMP (for example on macOS), and it does not hurt to add it, right?

anyway I don't mind closing this if you think it's not worth it.

Reply:
No, I didn't check the actual output (got sidetracked as usual), was making the comment so you would correct me if needed :-)